MetaMask/metamask-extension
# Update BannerAlert icon to match Figma Issue#20355

Fixes #20355
## Explanation
Updated BannerAlert to match figma icon (Warning -- Danger)
and ran " yarn jest --updateSnapshot " to update the snapshots 


## Screenshots/Screencaps

### Before
![image](https://github.com/MetaMask/metamask-extension/assets/56181880/06743694-75b6-4f3e-a04c-fa3a64adf679)

### After

![image](https://github.com/MetaMask/metamask-extension/assets/56181880/34abc60f-0e55-43c7-bb25-bad92e895db3)

## Manual Testing Steps
1)yarn run storybook
2) go to the components -- component library -- BannerAlert Docs -- see the icon change

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
